hal/src/b5som/network/quectel_ncp_client.cpp

```diff
@@ -695,6 +695,10 @@ int QuectelNcpClient::initReady() {
 
     muxerSg.dismiss();
 
+    // Make sure that we receive URCs only on AT channel, ignore response code
+    // just in case
+    CHECK_PARSER(parser_.execCommand("AT+QCFG=\"cmux/urcport\",1"));
```

Comment:
I remember that we were discussing about this command officially only supported on BG96. I do not see any reference in the EG91. Is it worth making sure this is officially supported on the EG91 just in case a future radio-version does not support it?

Reply:
For that case specifically, you can see that there is no check for the return OK status, so it's 'best effort': if it works, it works :)